resources/hu/cemu.po

```diff
@@ -932,44 +1081,51 @@ msgstr ""
 "A magasabb értékek a hang késését eredményezik, viszont kiküszöbölhetik a "
 "hangi problémákat, ha az emuláció túl lassú"
 
-#: GeneralSettings2.cpp:426
+#: GeneralSettings2.cpp
+msgid "Mono"
+msgstr "Monó"
+
+#: GeneralSettings2.cpp
 msgid "Stereo"
 msgstr "Sztereó"
 
-#: GeneralSettings2.cpp:428
+#: GeneralSettings2.cpp
+msgid "Surround"
+msgstr "Surround Sound"
```

Comment:
Why not just "Surround"? 🤔

Reply:
Hmm, this one isn't a recent string, so I don't really remember my reasoning behind it anymore, but I'll translate it properly now.